axelor-base/src/main/java/com/axelor/apps/base/service/AddressServiceImpl.java

```diff
@@ -341,4 +351,60 @@ public String getZipCode(Address address) {
     }
     return null;
   }
+
+  @Override
+  @Transactional(rollbackOn = {Exception.class})
+  public void setFormattedFullName(Address address) {
+        computeStringAddressTemplate(address);
+  }
```

Comment:
I understand why you did that but I think it's ok to just delete computeStringAddressTemplate and put everything directly in this method